src/Egret.Cli/Serialization/Audacity/AudacityImporter.cs

```diff
@@ -0,0 +1,118 @@
+            ImporterContext context)
+        {
+            if (context.Include.Filter is not null)
+            {
+                throw new Exception("The Audacity importer does not currently support include filters");
+            }
+
+            foreach (var path in resolvedSpecifications)
+            {
+                logger.LogTrace("Loading Audacity data file: {file}", path);
+
+                await using Stream stream = fileSystem.File.OpenRead(path);
+                var dataFile = serializer.Deserialize(stream, path);
+
+                var expectations = dataFile?.Tracks?.SelectMany((track, trackIndex) =>
+                        track.Labels.Select((label, labelIndex) =>
+                        {
+                            var temporalTolerance = context.Include.TemporalTolerance ?? defaultTolerance;
+                            var spectralTolerance = context.Include.SpectralTolerance ?? defaultTolerance;
+                            var overrideBounds = context.Include.Override;
+
+                            var name =
+                                $"Audacity Label {label.Title} in track {track.Name} ({trackIndex}:{labelIndex})";
+
+                            return label.IsSelPoint
+                                ? (IExpectation)new TemporalExpectation(label)
+                                {
+                                    Time = new TimeRange(
+                                        label.TimeStart.WithTolerance(temporalTolerance),
+                                        label.TimeEnd.WithTolerance(temporalTolerance)),
+                                    AnyLabel = new[] {label.Title},
+                                    Name = name,
+                                }
+                                : (IExpectation)new BoundedExpectation(label)
+                                {
+                                    Bounds = new Bounds(
+                                        overrideBounds?.Start ?? label.TimeStart.WithTolerance(temporalTolerance),
+                                        overrideBounds?.End ?? label.TimeEnd.WithTolerance(temporalTolerance),
+                                        overrideBounds?.Low ?? label.SelLow.WithTolerance(spectralTolerance),
+                                        overrideBounds?.High ?? label.SelHigh.WithTolerance(spectralTolerance)),
+                                    AnyLabel = new[] {label.Title},
+                                    Name = name,
+                                };
+                        }))
+                    .ToList() ?? new List<IExpectation> {new NoEvents()};
+
+                if (expectations.Count == 1 && expectations[0] is NoEvents)
+                {
+                    logger.LogWarning("No annotations found in {path}, producing a no events expectation", path);
+                }
+
+                if (context.Include.Exhaustive.HasValue && context.Include.Exhaustive.Value)
```

Comment:
```suggestion
                if (context.Include.Exhaustive is bool exhaustive))
```

Without this change the behaviour is not equivalent to the avianz importer. Why?

Because the config normalization does this later?